server.py

```diff
@@ -0,0 +1,352 @@
+        gen_order_id = fos.create_order(cust_id) 
+        order_obj = {
+            "customer_id": gen_order_id.cust_id,
+            "order_id": gen_order_id.order_id
+            }
+        return json.dumps(order_obj)
+
+    @app.route('/customer/add_food_to_order/<int:cust_id>', methods=['POST'])
+    def add_food_to_order(cust_id):
+        """
+        >> curl -H "Content-Type: application/json" -X POST -d "{\"order_id\":9, \"food_id\":1, \"food_qty\":1}" "localhost:8080/customer/add_food_to_order/2"
+        """
+        order_id = flask.request.json["order_id"]
+        food_id = flask.request.json["food_id"]
+        food_qty = flask.request.json["food_qty"]
+        add_food = fos.add_food_to_order(order_id, food_id, food_qty) 
+        order_obj = {
+            "order_id": add_food.order_id,
+            "food_id": add_food.food_id,
+            "food_qty": add_food.food_qty
+            }
+        return json.dumps(order_obj)
+
+    @app.route('/customer/update_food_to_order/<int:cust_id>', methods=['PUT'])
+    def update_process_order(cust_id):
+        """
+        >> curl -H "Content-Type: application/json" -X PUT -d "{\"order_id\":9, \"food_id\":1, \"food_qty\":2}" "localhost:8080/customer/update_food_to_order/2"
+        """
+        order_id = flask.request.json["order_id"]
+        food_id = flask.request.json["food_id"]
+        food_qty = flask.request.json["food_qty"]
+        fos.update_food_to_order(order_id, food_id, food_qty)
+        order_obj = {
+            "order_id": order_id,
+            "food_id": food_id,
+            "food_qty": food_qty
+            }
+        return json.dumps(order_obj)
+
+    @app.route('/customer/remove_food_to_order/<int:cust_id>', methods=['DELETE'])
+    def remove_food_to_order(cust_id):
+        """
+        >> curl -H "Content-Type: application/json" -XDELETE -d "{\"order_id\":9, \"food_id\":1}" "localhost:8080/customer/remove_food_to_order/2"
+        """
+        order_id = flask.request.json["order_id"]
+        food_id = flask.request.json["food_id"]
+        fos.remove_food_to_order(order_id, food_id)
+        order_obj = {
+            "order_id": order_id,
+            "food_id": food_id
+            }
+        return json.dumps(order_obj)
+           
+    @app.route('/customer/checkout/<int:cust_id>', methods=['PUT'])
+    def checkout(cust_id):
+        """
+        >> curl -H "Content-Type: application/json" -X PUT -d "{\"order_id\":9, \"order_address\":\"Karachi\"}" "localhost:8080/customer/checkout/2"
+        """
+        order_id = flask.request.json["order_id"]
+        order_status = "Checkedout"
+        order_address = flask.request.json["order_address"]
+        checkout_time = datetime.now()
+        delivery_time = timedelta(minutes=30)
+        estimated_time = checkout_time + delivery_time
+        total = fos.view_order_grand_total(order_id)
+        if total:
+            for cd, cosa, grand_total in total:
+                bill_amount = grand_total
```

Comment:
Since there's only one item in the `total` list, so you can just pick out the bill amount directly, no need for the for loop.

```python
     if total:
            bill_amount = total[0][2] # Verify this.
```

Reply:
change made